api/DOMMatrixReadOnly.json

```
@@ -2068,6 +2068,54 @@
           }
         }
       },
+      "translate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrixReadOnly/translate",
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true
```

I did a bit of digging and this shipped in Firefox 33: https://bugzilla.mozilla.org/show_bug.cgi?id=1018497

```suggestion
              "version_added": "33"
```